src/features/rbac/components/CreateProjectMemberButton.tsx

```diff
@@ -73,98 +75,116 @@ export function CreateProjectMemberButton(props: { projectId: string }) {
         email: values.email,
         role: values.role,
       })
-      .then(() => {
+      .then((result) => {
+        // @ts-expect-error (Due to unexpected error of ts)
+        if (result.senderId) {
+          setInviteNotification(true);
+          setTimeout(() => {
+            setInviteNotification(false);
+          }, 3000);
+        }
         form.reset();
         setOpen(false);
       })
       .catch(() => {
         form.setError("email", {
```

Comment:
let's display the actual errors here
- catch() - only console.error, do not form.setError
- use onError callback of mutCreateProjectMember to add the error state to the form with the real message. Check editDatasetItem for a reference implementation